app/presenters/error_message_translator.rb

```diff
@@ -69,7 +68,7 @@ def get_messages(translations, key, error)
   end
 
   def submodel_key_exists?(translations, key)
-    parent_model, _ = last_parent_attribute(translations,key)
+    parent_model, = last_parent_attribute(translations, key)
```

Comment:
Did this not mean...
    get two variables from `last_parent_attribute` and assign one to parent_model and ignore the second (_)?

would it make more sense to use 
`parent_model = last_parent_attribute(translations, key)[0]`
?